src/node_crypto.cc

```diff
@@ -1974,6 +1974,37 @@ static Local<Object> X509ToObject(Environment* env, X509* cert) {
 }
 
 
+// Put issuer inside the object
+static void AddIssuer(X509** cert,
+                      const STACK_OF(X509)* const peer_certs,
+                      Local<Object>* const info,
```

Comment:
Why is this a pointer?

Reply:
The reason was so that code in GetPeerCertificate depends on the value being pointed to by `info` be updated. But I realise that `info` does not have to be a pointer and we only need update it's pointer for this to work. Not sure if that made sense but I'll add a commit and hopefully this will become clearer. Thanks for pointing this out.